app/views/pages/get-started/build-your-first-app/sending-email-notifications.liquid

```diff
@@ -0,0 +1,211 @@
+---
+metadata:
+  title: Sending Email Notifications
+  description: Follow these step-by-step tutorials to build a simple To Do List application on platformOS.
+converter: markdown
+---
+
+platformOS offers an easy way of sending emails that could be used – for example – as notifications about a change in the application.
+
+## Enabling emails on the staging instance
+
+As a first step we would need to **enable sending emails** on the _staging_ instance. This step is not required on production instances, but the free _staging_ is restricted to sending emails only to a single address configured for testing. The emails send from those instances will always be redirected to the test address.
+
+To set the test email, we need to log in to the [Partner Portal](https://partners.platformos.com/) and choose the desired instance from the [Instances](https://partners.platformos.com/instances) list.
+
+Then, the _Update Configuration_ button will take us to the page where we can edit the desired values, but not before _Switching to edit mode_.
+
+What we seek here is a section of the configuration called `attributes` where we need to set two values – the `test_mode` and `test_email`. The first one is a boolean that we need to set to `true`, the second will be the test email address that all the emails for given instance will be send to:
+
+```javascript
+[...]
+"attributes": {
+  "test_mode": true,
+  "test_email": "your-email-address@example.com",
```

Comment:
per my comment above, this can also be `"test_email": "your-email-address@example.com,another@example.com",`